chart/skywalking/Chart.yaml

```diff
@@ -38,3 +38,7 @@ dependencies:
     version: 12.1.2
     repository: https://raw.githubusercontent.com/bitnami/charts/archive-full-index/bitnami
     condition: postgresql.enabled
+  - name: banyandb
+    version: 0.1.0
+    repository: file://../banyandb
+    condition: banyandb.enabled
```

Comment:
I think this doesn't work (?) when we release the skywalking Chart to Docker hub, using this installation method https://github.com/apache/skywalking-kubernetes#install-released-version-using-docker-helm-repository--430

Reply:
What should I change the repo to? Something like oci://registry-1.docker.io/apache/skywalking-banyandb-helm?